src/main/java/com/robertx22/age_of_exile/database/data/currency/CurrencyItem.java

```diff
@@ -52,10 +52,10 @@ public void appendHoverText(ItemStack stack, Level worldIn, List<Component> tool
 
 
         if (effect instanceof GearCurrency gc) {
-            if (gc.isAffectedByPotential()) {
```

Comment:
Same here, please don't rename my code, both are correct but changing tiny things like this will just lead to confusion in the future for me

Reply:
now I am confused about this, whether this method represents currency's effect won't **be affected** by potential, or represents the currency won't consume potential while using?